### ISSUE TITLE
importing libraries 

### ISSUE THREAD

**Issue by ZohraTagi** [spyder-ide/spyder#9994]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

Hi, i'm new in python. I want to import libraries such as numpy, matplotlib and pandas, but it doesn't show me any result about importing.

### Paste Traceback/Error Below (if applicable)

```python-traceback

runfile('C:/Users/Admin/.spyder-py3/untitled2.py', wdir='C:/Users/Admin/.spyder-py3')

runfile('C:/Users/Admin/.spyder-py3/untitled2.py', wdir='C:/Users/Admin/.spyder-py3')

```

## Versions

* Spyder version: 
* Python version: 
* Qt version: 
* PyQt version: 
* Operating System name/version: 

### Dependencies

```
PASTE DEPENDENCIES HERE
```

**Comment by ccordoba12** [comment]
Importing a library doesn't do anything per se. You need to do something with, for instance the following code will compute the absolute value of all numbers in a list:

```python
import numpy as np

li = [2, -1, 3, -5]

print(np.abs(li))
```

I'm closing this because it's a general Python question, not a problem with Spyder (which this place is used for).

Please ask additional questions here:

https://stackoverflow.com/

**Comment by ZohraTagi** [comment]
thank you for your response.
I have additional question. I have a problem with nan values. I want to replace nan values form my dataset, for doing it i used 

from sklearn.preprocessing import Imputer
imputer= Imputer(missing_values='np.nan', strategy="mean", axis=0)
imputer=imputer.fit(X[:, 1:3])
X[:, 1:3]=imputer.transform(X[:, 1:3])

this code, but does not work.
Can you tell me what is wrong?

**Comment by ccordoba12** [comment]
> I have additional question

As I clearly said above: please ask additional questions in Stackoverflow.

https://stackoverflow.com/
